lib/stripe/stripe_object.rb

```diff
@@ -249,7 +256,24 @@ def method_missing(name, *args)
     end
 
     def respond_to_missing?(symbol, include_private = false)
-      @values && @values.has_key?(symbol) || super
+      if symbol.to_s.end_with?('?')
+        name = chop_symbol(symbol)
+        has_boolean_value?(name) || super
+      else
+        @values && @values.has_key?(symbol) || super
+      end
     end
+
+    # Returns a new symbol with the last character removed.
+    # e.g. `:foo=` becomes `:foo`.
+    def chop_symbol(symbol)
+      symbol.to_s[0...-1].to_sym
+    end
+
+    # Returns true if the value exists and is a boolean.
+    def has_boolean_value?(symbol)
+      @values.has_key?(symbol) && !!@values[symbol] == @values[symbol]
```

Comment:
Also bikeshed: what do you think of `[TrueClass, FalseClass].include?(@values[symbol].class)`? I find it's generally a little easier to read for language newcomers.